.github/workflows/build.yml

```
@@ -65,6 +65,11 @@ jobs:
           name: packages-${{ matrix.os_name }}
           path: ./artifacts/packages
 
+      - name: Publish pre-release NuGet packages to GitHub packages
+        run: dotnet nuget push "artifacts\packages\*.nupkg" --api-key ${{ secrets.GITHUB_TOKEN }} --skip-duplicate  --source https://nuget.pkg.github.com/justeat/index.json
+        if: ${{ github.repository_owner == 'justeat' && runner.os == 'Windows' }}
+        continue-on-error: true
```

I think this would be neater than just ignoring the error:

```suggestion
        if: ${{ github.repository_owner == 'justeat' && github.ref == 'refs/heads/master' && runner.os == 'Windows' }}
```